documentation/adrs/README.md

```diff
@@ -0,0 +1,55 @@
+# Architecture Decision Records
+
+ADRs are automatically published to our Log4brains architecture knowledge base:
+
+**http://INSERT-YOUR-LOG4BRAINS-URL**
```

Comment:
do we have a valid URL?

Reply:
I'm going to delete this page since is no longer update with the way we build the ADRs